django_mongodb/expressions.py

```diff
@@ -79,8 +92,63 @@ def order_by(self, compiler, connection):
     return self.expression.as_mql(compiler, connection)
 
 
-def query(self, compiler, connection):  # noqa: ARG001
-    raise NotSupportedError("Using a QuerySet in annotate() is not supported on MongoDB.")
```

Comment:
You deleted this but left it in the README and didn't remove any of the skips? Is it meant to be solved in this PR?

Reply:
I've removed skipped test from feature.py, is there any other source of subquery test that I forgot?

Reply:
Yes, but I think that I modify both things. is something there that I am not seeing?

Reply:
Added Queryset unit tests.